src/mbgl/gfx/offscreen_texture.hpp

```diff
@@ -17,6 +17,8 @@ class OffscreenTexture : public gfx::Renderable {
 public:
     virtual ~OffscreenTexture() = default;
 
+    virtual bool isRenderable() = 0;
```

Comment:
nit: const method

Reply:
Well, technically the isRenderable check binds the offscreentexture, so I'm not sure I'd consider it `const`